dist/tools/ci/can_fast_ci_run.py

```diff
@@ -164,14 +164,37 @@ def classify_changes(riotbase=None, upstream_branch="master"):
             match = REGEX_GIT_DIFF_SINGLE_FILE.match(line)
             if match:
                 file = match.group(1)
-                change_set.add_file(file)
+
+                if only_comment_change(file, upstream_branch) is False:
+                    change_set.add_file(file)
```

Comment:
```suggestion
                if only_comment_change(file, upstream_branch):
                    change_set.add_file_force_classification_as_doc(file)
                else:
                    change_set.add_file(file)
```

Maybe it makes sense to still classify the change, but force categorizing it as comment change? We might decide that e.g. on a change that only touches files in `dist/tools` even the static tests are skipped or similar, but that would require all doc changes to be caught.

(There currently is no `add_file_force_classification_as_doc` or similar implemented.)

Reply:
yup. let's do that as folo.
It would also be nice if the comment-only files would be listed in the "json" output.